core/src/main/java/apoc/SystemLabels.java

```diff
@@ -8,5 +8,6 @@
     Procedure,
     Function,
     ApocUuid,
-    ApocTrigger
+    ApocTrigger,
+    VRCatalog
```

Comment:
```suggestion
    DataVirtualizationCatalog
```

Reply:
Or rename to `VirtualResourceCatalog` and rename the `apoc.dv` package to `apoc.vr` in that case